src/components/goodid/GoodIdOnboard.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback, useContext } from 'react'
+import { View } from 'react-native'
+import { OnboardScreen } from '@gooddollar/good-design'
+import { GoodWalletContext } from '../../lib/wallet/GoodWalletProvider'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import { withStyles } from '../../lib/styles'
+
+const GoodIdOnboardImpl = ({ screenProps, styles }) => {
+  // isValid is result from FV
+  const { navigateTo, isValid } = screenProps
+  const { goodWallet } = useContext(GoodWalletContext)
+
+  const navigateToFV = useCallback(() => {
+    navigateTo('FaceVerificationIntro')
+  }, [navigateTo])
+
+  // passing isValid=true to Claim screen will automatically trigger the claim
+  const onSkip = useCallback(() => {
+    navigateTo('Claim', { isValid })
```

Comment:
isValid won't be needed here

Reply:
it could be btw..

If checkAvailableOffers will also be handled in the onboard-screen. 

Then the flow would be: FV > SegmentationScreen > Onboard > (CheckAvailableOffers, yes > offers, no > claim)